tests/deprecations_ignore.yml

```diff
@@ -56,17 +56,7 @@
 
 
 # CLI
-- tests/cli/commands/test_connection_command.py::TestCliAddConnections::test_cli_connection_add
-- tests/cli/commands/test_connection_command.py::TestCliImportConnections::test_cli_connections_import_should_load_connections
-- tests/cli/commands/test_connection_command.py::TestCliImportConnections::test_cli_connections_import_should_not_overwrite_existing_connections
-- tests/cli/commands/test_connection_command.py::TestCliImportConnections::test_cli_connections_import_should_overwrite_existing_connections
 - tests/cli/commands/test_kubernetes_command.py::TestGenerateDagYamlCommand::test_generate_dag_yaml
```

Comment:
This should be decoupled from the core first. `airflow kubernetes` right now live into the core but use into the provider

Reply:
Not fully decouple from the core, just do the same thing that we done for the celery.
When it depend on the installed version of the provider, if newer specific version than use from the provider otherwise use from the core

Reply:
Yes. created an issue for that https://github.com/apache/airflow/issues/39199